python/security/credentials/json.py

```diff
@@ -0,0 +1,25 @@
+import json
+from json import JSONDecodeError
+
+from config.security.json import SECRET_JSON_PATH
+from exceptions import ConfigurationError
+from security.credentials.base import CredentialsReader
+
+
+class JSONCredentialsReader(CredentialsReader):
+    def __init__(self):
+        try:
+            self.json = json.load(open(SECRET_JSON_PATH, "r"))
+        except FileNotFoundError as e:
+            raise ConfigurationError(f"No credentials JSON found at {SECRET_JSON_PATH}.") from e
+        except JSONDecodeError as e:
+            raise ConfigurationError(f"File {SECRET_JSON_PATH} is not a valid JSON file") from e
+
+    def read(self, path: str, fields: tuple = None) -> dict:
+        entry = self.json
+        try:
+            for key in path.replace("\\", "/").split("/"):
```

Comment:
Why do you need to replace double back-slashes by slashes ?
If it is useful, please add a comment because it seems not clear enough :-)